jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -396,6 +400,13 @@ public Object run() {
  */
 @CallerSensitive
 public static Class<?> forName(String className) throws ClassNotFoundException {
+	return forName(className, null);
+}
+
+/*[IF JAVA_SPEC_VERSION >= 18]*/
+@CallerSensitiveAdapter
```

Comment:
`getStackClass()` checks the annotation `@CallerSensitive`, now `@CallerSensitiveAdapter` need to be checked instead.

Reply:
updated.